proto/stat.proto

```diff
@@ -410,4 +411,8 @@ message GetStatDateWithinPlaceRequest {
 message GetStatDateWithinPlaceResponse {
   // Keyed by stat var.
   map<string, StatDateList> data = 1;
-}
\ No newline at end of file
+}
+
+message Categories {
+  repeated string category = 1;
+}
```

Comment:
should this also be in place.proto?

Reply:
The response object used right now is in internal.proto, so have changed GetPlacePageDataResponse there.  Is there a specific place that you have in mind in place.proto that we should change?

Reply:
Yea, maybe this can also be in internal.proto. "stat.proto" should contain proto for stat related API. The folder is more or less based on the API functional grouping.

Reply:
Done